tests/acceptance/premium/test_branches.py

```diff
@@ -120,3 +120,36 @@ def test__allow_more_than_one_user_by_ids(self, project, branch, make_user):
             ]
         )
         assert merge_access_user_ids == []
+
+    def test__allow_exactly_one_user_id(self, project, branch, make_user):
```

Comment:
While running this test locally I noticed that it randomly fails with:
```
(...) code 422 & body: '{"message":["Push access levels user is not a member of the project"]}'
```

It looks like this GitLab issue https://gitlab.com/gitlab-org/gitlab/-/issues/383666 - I'll try to bump it then.